### ISSUE TITLE
Deprecation torchsde version

### ISSUE THREAD

**Issue by moseti1** [google-research/torchsde#134]
DEPRECATION: torchsde 0.2.5 has a non-standard dependency specifier numpy>=1.19.*; python_version >= "3.7". pip 23.3 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of torchsde or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at

**Comment by marshalleq** [comment]
I'm getting this too, very annoying.

**Comment by Exohayvan** [comment]
I fixed mine by doing:
```
pip install numpy==1.23.0
pip install protobuf==3.20.3
pip install trainer --upgrade
pip install tts --upgrade
```

not sure if this will help anyone but that’s what worked for me

**Comment by RandomArray** [comment]
---

**Quick Fix:** For an immediate resolution to this issue, find the METADATA file in the package's dist-info folder (e.g., `torchsde-0.2.5.dist-info`), and edit the numpy version number on line 18, removing the trailing `.*`. It should be modified from `1.19.*` to `1.19`.

---

Now, for those interested in the whole process:

Use `pip show torchsde` to find out where the package is. You'll see a path next to `Location:`; that's where your package is. Be aware, this path might be different if you're using a virtual environment.

```shell
(comfy) c:\ComfyUI_windows_portable\update>pip show torchsde
Name: torchsde
Version: 0.2.5
Summary: SDE solvers and stochastic adjoint sensitivity analysis in PyTorch.
Home-page: https://github.com/google-research/torchsde
Author: Xuechen Li
Author-email: lxuechen@cs.toronto.edu
License: UNKNOWN
**Location: c:\users\mikej\appdata\roaming\python\python310\site-packages**
Requires: boltons, numpy, scipy, torch, trampoline
Required-by:
```

I tried changing something in the appdata folder but ran into the same problem. Trying to make ComfyUI work, I noticed when running their updater that packages were located in `c:\comfyui_windows_portable\python_embeded\lib\site-packages`. I found the `torchsde-0.2.5.dist-info` folder there, and updated the METADATA file by removing the `.*` from the end of the numpy version. Changed it from `1.19.*` to `1.19`.

The updated METADATA File:

```text
Metadata-Version: 2.1
Name: torchsde
Version: 0.2.5
Summary: SDE solvers and stochastic adjoint sensitivity analysis in PyTorch.
Home-page: https://github.com/google-research/torchsde
Author: Xuechen Li
Author-email: lxuechen@cs.toronto.edu
License: UNKNOWN
Platform: UNKNOWN
Classifier: Programming Language :: Python :: 3
Classifier: License :: OSI Approved :: Apache Software License
Requires-Python: ~=3.6
Requires-Dist: boltons (>=20.2.1)
Requires-Dist: torch (>=1.6.0)
Requires-Dist: trampoline (>=0.1.2)
Requires-Dist: numpy (==1.19.*) ; python_version < "3.7"
Requires-Dist: scipy (==1.5.*) ; python_version < "3.7"
Requires-Dist: numpy (>=1.19) ; python_version >= "3.7"
Requires-Dist: scipy (>=1.5) ; python_version >= "3.7"

UNKNOWN

```

**Comment by moseti1** [comment]
I commented out the two lines showing NumPy Requirements, and my problems seems to have gone away for now.

**Comment by jphan32** [comment]
This issue has been resolved and merged in PR #133.
